src/db/schemas/expense.model.ts

```diff
@@ -23,19 +23,22 @@ const expenseModel = pgTable("expense", {
     .$defaultFn(() => Bun.randomUUIDv7())
     .primaryKey()
     .notNull(),
-
   payerId: varchar({ length: 60 })
     .notNull()
     .references(() => userModel.id, {
       onDelete: "cascade",
       onUpdate: "cascade",
     }),
-  categoryId: varchar({ length: 60 })
+  creatorId: varchar({ length: 60 })
```

Comment:
This is good, we can store who created the expense who paid for that expense, and split across the remaining people for the split.
We just need to let anyone choose the payerId and not just admin

Reply:
Good idea. Now user can create expense on behalf of another payer.